### PR TITLE
Eh team collection

### PR DESCRIPTION
**Pull request by Shock253**
These classes should be able to work standalone. TeamCollection should handle all of the loading and storing of the teams.csv data

### CODE REVIEW

File: test/team_collection_test.rb [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+  end
+
+  def test_csv_loads
+    team_collection = TeamCollection.new("./data/teams.csv")
+
+    team_a = team_collection.teams.first
+    team_z = team_collection.teams.last
+
+
+    assert_equal 1,                       team_a.team_id
```
Comment (javier-aguilar) [inline test/team_collection_test.rb:28]: is the spacing intentional on the assertions?

**Comment by javier-aguilar** [comment]
Everything looks good. Nice use of the mock and stubs

File: lib/team.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Team
+  attr_reader :team_id,
+              :franchiseId,
+              :teamName,
+              :abbreviation,
+              :stadium,
+              :link
+
+  def initialize (team_params)
+    @team_id        = team_params[:team_id].to_i
```
Comment (steve-meyers) [inline lib/team.rb:10]: At some point maybe we can change all these to snake_case for consistency

File: lib/team_collection.rb [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+  end
+
+  def create_teams(csv_file_path)
+    csv = CSV.read(csv_file_path, headers: true, header_converters: :symbol)
+    csv.map do |row|
+       Team.new(row)
+    end
+  end
+
+  def find_team_by_id(id)
```
Comment (steve-meyers) [inline lib/team_collection.rb:19]: I might be able to use this to speed up my methods

File: test/team_collection_test.rb (hunk 1 above)
Comment (steve-meyers) [reply test/team_collection_test.rb:28]: It looks like it is an organizational approach but it looks a little strange to me

**Review by steve-meyers — APPROVED** [review]
I think everything looks good!

**Pull request merged**